Summation of tree changes

Krumel opened the pull request:
I've added a summation of all the changes to attributes, to the tooltip that appears on hovering over a node, when clicking that node would change multiple nodes in the tree (adding or removing).

It's a bit clunky, but I think it would be a nice feature to have, as you can quickly compare clusters of nodes.

It also doesn't have any form of localization support atm, but so does the tooltip line directly above.

brather1ng commented:
I like the idea, I think it came up before already. However, I don't think it should always be on. It would need to be a toggle to not clutter the tooltip.

EmmittJ commented:
Yeah, this is something I had done before, but I never really finished the
idea. It definitely needs a menu option.

On Nov 22, 2016 6:27 AM, "brather1ng" <notifications@github.com> wrote:

> I like the idea, I think it came up before already. However, I don't think
> it should always be on. It would need to be a toggle to not clutter the
> tooltip.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/EmmittJ/PoESkillTree/pull/402#issuecomment-262254109>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHIbbbuAX8xacx_2Igd4lsEEJoX-HRM2ks5rAvu6gaJpZM4K48Ml>
> .
>

Krumel commented:
I've changed it to only show while pressing Ctrl.

EmmittJ commented:
So, I had some talk with @brather1ng, and we aren't sure ctrl is the best option (since ctrl + click is for zooming out). What about adding it to the menu as well as giving it a hotkey? This way you can toggle it on/off pretty fast without having to hold down a key.

Krumel commented:
Sure, I'll change it that way, it'll just take a bit of time for me to get into the code some more.
Is there any key you would prefer, or should I just pick one (or is there a system for rebindable keys)?

MLanghof commented:
...you bound it to what most programs use as "Search" hotkey?

Krumel commented:
It's an easily reached hotkey and not yet bound. 
And that's why I asked for they key I should bind it to,  I'm shit at choosing

SpaceOgre commented:
What about `Ctrl + g` as in Group? I agree that `Ctrl + f` is not the best. I think I will bind that to the search-box actually :)

brather1ng commented:
Seems to work fine 👍 

I think the menu entry would fit better under View, but that's just my opinion.

Krumel commented:
Okay, I've added both those suggestions in.
I've also restructured the code a bit to make githubs diff spazz out less and to make it easier to see what I changed.

EmmittJ commented:
Sweet! Looks good. I am going to merge this now!
